### PR TITLE
feature: add commision on No BS new and renewal

### PR DESCRIPTION
**Pull request by cheedinmar**

### CODE REVIEW

**Comment by cheedinmar** [comment]
work stil in progress

File: src/components/PaymentLog.vue [hunk 1]
```diff
@@ -895,6 +908,27 @@ export default {
...
+    watchSalesCategory(){ 
+      if(this.isAltaraPay){
+        if(this.salesLogForm.sales_category_id == 9 ){
+        this.commission.status = true
+        //if sales-category is "NoBS"
+        this.businessTypes = this.biz_type.filter((business_type)=>{
+          //return only this business type, 
+          return business_type.name.includes('No BS')
+        })
+        this.salesLogForm.business_type_id.id == 16
```
Comment (AltaraNg) [inline src/components/PaymentLog.vue:920]: can we make use of slug instead of id?

File: src/components/PaymentLog.vue [hunk 2]
```diff
@@ -1135,17 +1168,34 @@ export default {
...
         this.fPayment = actualDownpayment
         this.rPayment = rePayment
         this.pPrice = total
         this.test1 = false
         const months = this.rDuration / 30
-
         const cycle = Math.ceil(28 / this.repaymentCircle)
         const additionalRepayment = this.rPayment / (months * cycle)
+        
+        if(this.isAltaraPay && this.salesLogForm.sales_category_id == 9){
```
Comment (AltaraNg) [inline src/components/PaymentLog.vue:1182]: let's use slug incase id changes

File: src/components/PaymentLog.vue [hunk 3]
```diff
@@ -1135,17 +1168,34 @@ export default {
...
+          //check if on altara pay and New BS sales category
+          console.log('hello 1');
+          
+           if(this.salesLogForm.business_type_id.id == 16 || this.salesLogForm.business_type_id.id ==  15){
```
Comment (AltaraNg) [inline src/components/PaymentLog.vue:1186]: same here

File: src/components/PaymentLog.vue [hunk 4]
```diff
@@ -1135,17 +1168,34 @@ export default {
...
+          //if biz-type is BS-new customer
+          this.commission.percentage = "6%"
+          this.commission.amount =  this.pPrice * (6/100)
+          
+          //add a 6% commision on the downpayment
+        }else if(this.salesLogForm.business_type_id.id == 17 || this.salesLogForm.business_type_id.id ==  18){
```
Comment (AltaraNg) [inline src/components/PaymentLog.vue:1192]: same here

File: src/components/PaymentLog.vue [hunk 5]
```diff
@@ -1209,6 +1259,12 @@ export default {
 
         const unwrapped = fetchGetCalclations.data.data
         this.calculation = unwrapped
+        
+        
+        const filter = this.calculation.filter((obj)=>{
+          return obj.business_type_id >=15
+        })
+        console.log(filter);
```
Comment (AltaraNg) [inline src/components/PaymentLog.vue:1267]: remove console

File: src/components/PaymentLog.vue [hunk 6]
```diff
@@ -1135,17 +1168,34 @@ export default {
...
+          //if biz-type is BS-new customer
+          this.commission.percentage = "6%"
+          this.commission.amount =  this.pPrice * (6/100)
+          
+          //add a 6% commision on the downpayment
+        }else if(this.salesLogForm.business_type_id.id == 17 || this.salesLogForm.business_type_id.id ==  18){
+          //BS-renewal
+          this.commission.percentage = "3%"
+          this.commission.amount =  this.pPrice * (3/100)
+          console.log(this.commission.amount, 'renewal customer');
```
Comment (AltaraNg) [inline src/components/PaymentLog.vue:1196]: remove console

**Pull request closed without merging**